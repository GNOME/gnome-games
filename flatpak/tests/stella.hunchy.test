[Retro Reftest]
Path=/stella/hunchy
Core=/app/lib/libretro/stella_libretro.so
Medias=hunchy.a26;

[Frame 1]
Tests=Run;Video;
Video=stella.hunchy.1.png
