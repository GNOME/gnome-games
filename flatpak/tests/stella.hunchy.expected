/stella/hunchy/Boot: OK
/stella/hunchy/1/FastForward: OK
/stella/hunchy/1/Run: OK
/stella/hunchy/1/Video: OK
